btn-1cc-checkout.js

```diff
@@ -43,47 +43,153 @@ function btnCheckout(){
     });
   }
 
+  // For attaching event listener to Woodmart's sticky add-to-cart
+  document.addEventListener('scroll',(e)=>{
+     
+    var stickyBtn=document.querySelectorAll('#btn-1cc-pdp')[1];
```

Comment:
@angbpy This var will be null for all other pages except the product display page. Please handle this case